tardis/model.py

```diff
@@ -207,7 +207,7 @@ def calculate_j_blues(self, init_detailed_j_blues=False):
             logger.info('Calculating J_blues for radiate_rates_type=detailed')
 
             self.j_blues = pd.DataFrame(
-                self.j_blue_estimators.transpose() *
+                self.j_blue_estimators *
                 self.j_blues_norm_factor.value,
```

Comment:
There are people that fancy the operator on the next line for that reason.
(Which is against PEP)

The pythonic way might be to save the calculation in a temporary variable
and pass that as the argument.

Wolfgang Kerzendorf <notifications@github.com> schrieb am Di., 16. Feb.
2016 17:11:

> In tardis/model.py
> <https://github.com/tardis-sn/tardis/pull/486#discussion_r53032635>:
>
> > @@ -207,7 +207,7 @@ def calculate_j_blues(self, init_detailed_j_blues=False):
> >              logger.info('Calculating J_blues for radiate_rates_type=detailed')
> >
> >              self.j_blues = pd.DataFrame(
> > -                self.j_blue_estimators.transpose() *
> > +                self.j_blue_estimators *
> >                  self.j_blues_norm_factor.value,
>
> You are the style person - can you suggest how to make it PEP8 conform?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tardis-sn/tardis/pull/486/files#r53032635>.
>